### PR TITLE
PR with split view comments

### PR DESCRIPTION
**Pull request by cheap-glitch**
This PR is meant to test edge cases in `center-reactions-popup.css`.

### CODE REVIEW

File: play with this file, not readme [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-1
```
Comment (cheap-glitch) [inline play with this file, not readme:2]: This is a comment on the *left* side of the split view.

File: play with this file, not readme [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+2
```
Comment (cheap-glitch) [inline play with this file, not readme:2]: This is a comment on the *right* side of the split view.
